new-integration-tests/src/test/java/oracle/weblogic/kubernetes/assertions/impl/Domain.java

```diff
@@ -85,19 +88,57 @@ public static boolean doesCrdExist() throws ApiException {
     };
   }
 
-  public static boolean adminT3ChannelAccessible(String domainUid, String namespace) {
-    return true;
-  }
+  /**
+   * Verify admin node port(default/t3channel) is working by accessing REST endpoint using
+   * the node port.
+   *
+   * @param nodePort admin default/t3channel node port
+   * @param username admin user name to access the REST url
+   * @param password admin password to access the REST url
+   * @return true if admin REST url is accessible using node port
+   */
+  public static boolean adminNodePortAccessible(int nodePort, String username, String password) {
+    StringBuffer cmd = new StringBuffer();
+    cmd.append("curl --silent --show-error --noproxy ")
```

Comment:
we should move away from curl and start using java apis
https://openjdk.java.net/groups/net/httpclient/intro.html

Reply:
sure, will make the changes in separate PR.